Tools/sitl_run.sh

```diff
@@ -99,10 +99,19 @@ for file in "$@"; do
 done
 
 export PX4_SIM_MODEL=${model}
+export PX4_SIM_PROGRAM=${program}
 
 SIM_PID=0
 
-if [ "$program" == "jmavsim" ] && [ ! -n "$no_sim" ]; then
+if [ "$program" == "sih" ] && [ ! -n "$no_sim" ] && [[ ! -n "$HEADLESS" ]]; then
+	# Start Java simulator for displaying sih
+	if [ "$model" == "plane" ]; then
```

Comment:
Not related to this PR, but more of a general question - Does this mean that currently SIH only supports either a plane / multicopter?

Reply:
yes, and tailsitter but the [PR](https://github.com/PX4/PX4-Autopilot/pull/18162) is not merged yet.

Reply:
Now, I don't want to use the exact `iris` and `plane` since the sih models are quite different and will require other parameters to fly correcty (the iris flies correctly but that's just a coincidence.

Reply:
May I ask why we do not want to support the default airframes for SIH?

Reply:
@Jaeyoung-Lim the default airframes are different from the airframes in sih.
The `plane` doesn't have the same mixer as the `airplane` for instance. The sih `airplane` is AERT and the gazebo `plane` is RTFAlArEF (Rudder-Throttle-Flap-left Aileron-right Aileron-Elevator-Flap). I don't really want to modify the sih to match was is in place in gazebo. Does that make sense to have different airframes then?

Reply:
> I don't really want to modify the sih to match was is in place in gazebo. Does that make sense to have different airframes then?

What would we lose from modifying the SIH airframe? I am not against having an airframe that works better in SIH, but I think it is best if we can have the default model also work with SIH. This way we can also see the different behaviors in different simulations with the same airframe.